ci-operator/step-registry/odf/apply-storage-cluster/odf-apply-storage-cluster-commands.sh

```
@@ -42,7 +42,7 @@ sleep 60
 
 echo "⏳ Wait for StorageCluster to be deployed"
 oc wait "storagecluster.ocs.openshift.io/ocs-storagecluster"  \
-    -n $ODF_INSTALL_NAMESPACE --for=condition='Available' --timeout='10m'
+    -n $ODF_INSTALL_NAMESPACE --for=condition='Available' --timeout='15m'
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)